tests/data/bundle.yaml

```diff
@@ -12,21 +12,9 @@ applications:
 relations:
 #######################################
 #  Kubeapi-load-balancer relations
-- - kubeapi-load-balancer:apiserver
-  - kubernetes-control-plane:kube-api-endpoint
-- - kubernetes-control-plane:loadbalancer-external
-  - kubeapi-load-balancer:lb-consumers
-- - kubernetes-control-plane:loadbalancer-internal
-  - kubeapi-load-balancer:lb-consumers
-- - kubeapi-load-balancer:certificates
-  - easyrsa:client
+- [kubernetes-control-plane:loadbalancer-external, kubeapi-load-balancer:lb-consumers]
+- [kubernetes-control-plane:loadbalancer-internal, kubeapi-load-balancer:lb-consumers]
+- [kubeapi-load-balancer:certificates,             easyrsa:client]
 #######################################
 #  keepalived relations
-- - keepalived:juju-info
-  - kubeapi-load-balancer:juju-info
-- - keepalived:lb-sink
```

Comment:
WHy are these relations getting removed?

Reply:
well, mostly b/c they are unnecessary of the test, and secondarily -- charms like kubernetes-control-plane and kubernetes-worker no longer suppose these endpoints.